app/apps/server/bridges/messages.js

```diff
@@ -48,12 +48,16 @@ export class AppMessageBridge {
 
 		const msg = this.orch.getConverters().get('messages').convertAppMessage(message);
 
-		Notifications.notifyUser(user.id, 'message', Object.assign(msg, {
+		if (!msg) {
+			return;
+		}
+
+		Notifications.notifyUser(user.id, 'message', {
+			...msg,
 			_id: Random.id(),
 			ts: new Date(),
-			u: undefined,
-			editor: undefined,
-		}));
+			u: msg.u ? msg.u : Users.findOneByAppId(appId) || undefined,
```

Comment:
I wonder if the handling of the app's user shouldn't be done in the Apps-Engine, similarly to how it is handled in message sending. It does look like the responsibilities of the bridges should be to simply receive instructions and apply them to Rocket.Chat, and handling this use case in here seems to be outside of that scope